specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2024-09-10-preview/HybridCompute.json

```diff
@@ -2771,6 +2966,154 @@
+          "description": "The URI to fetch the next page of extension metadata."
+        }
+      },
+      "description": "The List Extension Metadata response."
+    },
+    "ExtensionValueV2": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ExtensionValueV2Properties",
+          "description": "The single extension based on search criteria."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Describes an Extension Metadata."
+    },
+    "ExtensionValueV2Properties": {
+      "type": "object",
+      "properties": {
+        "extensionUris": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string",
+            "format": "uri"
+          },
+          "description": "A list of locations where the extension packages can be found."
+        },
+        "extensionSignatureUri": {
+          "type": "string",
+          "readOnly": true,
+          "format": "uri",
+          "description": "Location of the signature files for the extension."
+        },
+        "operatingSystem": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The operating system (Windows, Linux, etc.) this extension supports."
+        },
+        "architecture": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          },
+          "description": "Architectures (x64, arms64, etc.) that this extension supports."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ExtensionValueProperties"
+        }
+      ],
+      "description": "Describes Extension Metadata properties."
+    },
+    "ExtensionTypeListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ExtensionType"
+          },
+          "description": "The list of extension types.",
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of extension types."
+        }
+      },
+      "description": "The List of Extension Types"
+    },
+    "ExtensionType": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ID of the extension type."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the extension type."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the extension type."
+        }
+      },
+      "description": "Describes an Extension Type."
```

Comment:
Please use the trackedresource from commontypes for this

Reply:
Removed this property since we don't want this to be a tracked resource.